src/service/resources-impl.js

```diff
@@ -686,9 +686,8 @@ export class Resources {
 
     if (this.documentReady_ && this.firstPassAfterDocumentReady_) {
       this.firstPassAfterDocumentReady_ = false;
-      this.viewer_.postDocumentReady(this.viewport_.getSize().width,
-        this.win.document.body./*OK*/scrollHeight);
-      this.updateScrollHeight_();
```

Comment:
I've checked Google's viewer code, we don't use `documentResized` event at all, and we only use the title sent in `documentLoaded`.